packages/react-core/src/components/Select/Select.tsx

```diff
@@ -78,6 +78,8 @@ export interface SelectProps extends MenuProps, OUIAProps {
   maxMenuHeight?: string;
   /** Indicates if the select menu should be scrollable */
   isScrollable?: boolean;
+  /** Flag indicating if scroll on focus of the first menu item should occur. */
+  shouldpreventScrollOnItemFocus?: boolean;
```

Comment:
Small casing issue in all of the changed files:

```suggestion
  shouldPreventScrollOnItemFocus?: boolean;
```

Reply:
Oh silliness! Derp!